examples/docs_snippets/docs_snippets/guides/dagster/graph_job_op/graph_with_config_mapping.py

```diff
@@ -1,27 +1,27 @@
-from dagster import ConfigMapping, graph, op
+from dagster import ConfigMapping, job, op
 
 
 @op(config_schema={"param": str})
 def do_something(_):
     ...
 
 
-@graph
+# start
+config_mapping = ConfigMapping(
```

Comment:
If we're not going to include the ConfigMapping inline with the job definition, probably prettier to use the decorator.